### ISSUE TITLE
Lockup module "total-locked-of-denom" passes in wrong value by default

### ISSUE THREAD

**Issue by czarcas7ic** [osmosis-labs/osmosis#670]
As an example, if you try:

```bash
osmosisd query lockup total-locked-of-denom gamm/pool/2
```

You will see `Error: time: unknown unit "d" in duration "1d"`

You must instead pass in the min duration (in seconds) for it to work

```bash
osmosisd query lockup total-locked-of-denom gamm/pool/2 --min-duration "1209600s"
```

Also the `osmosisd query lockup total-locked-of-denom -h` shows to pass the duration in days which, as explained above, does not work.

Obviously not a huge deal, just struggled with it while writing the lockup module documentation

**Comment by ValarDragon** [comment]
Does this query even work right now? I thought it was broken b/c we disabled accumulation store. (Hopefully it comes back in next major upgrade, its pending an IAVL fix)

cc @mattverse

**Comment by czarcas7ic** [comment]
I just went back and tried to manually calculate it (used the module-locked-amount to see how much gamm/pool/2 is locked in total, then used the total-locked-of-denom for each of the three time periods) and they did not equal each other. Still weird that total-locked-of-denom is still outputting numbers. I mean of course people could have locked gamm in time periods other than the three main locking periods, but my numbers were off by a lot. I will do more digging.

@ValarDragon If you could let me know of any other module functions that you are positive are broken, that would help me while I finishing the documentation. I will leave them blank for now so I don't get caught up in something I can't fix. Thank you!

**Comment by ValarDragon** [comment]
Closed by #859
